include/umf/memory_pool_ops.h

```diff
@@ -191,6 +191,25 @@ typedef struct umf_memory_pool_ops_t {
     ///         failure.
     ///
     umf_result_t (*ext_trim_memory)(void *pool, size_t minBytesToKeep);
+
+    ///
+    /// @brief Post-initialization hook for deferred setup.
+    ///
+    /// \details
+    /// * Called by UMF **after** @ref initialize during umfPoolCreate().
+    /// * Between @ref initialize and @ref ext_post_initialize there **may** be
+    ///   multiple calls to @ref ext_ctl for early configuration.
```

Comment:
It is unclear how pool/provider developer should decide how initialization steps should be split between `initialize` and `ext_post_initialize`

Reply:
Honestly, I think it is an error-prone approach